azure-tools-common/src/main/java/com/microsoft/azure/maven/function/handlers/FunctionCoreToolsHandlerImpl.java

```diff
@@ -28,25 +29,23 @@
     public static final String GET_LOCAL_VERSION_FAIL = "Failed to get Azure Functions Core Tools version locally";
     public static final Version LEAST_SUPPORTED_VERSION = Version.valueOf("2.0.1-beta.26");
 
-    private AbstractFunctionMojo mojo;
     private CommandHandler commandHandler;
 
-    public FunctionCoreToolsHandlerImpl(final AbstractFunctionMojo mojo, final CommandHandler commandHandler) {
-        this.mojo = mojo;
+    public FunctionCoreToolsHandlerImpl(final CommandHandler commandHandler) {
```

Comment:
Do we need to add another constructor which add funcPath as parameter?

Reply:
This PR is mainly for file move, the changes in this java file is due to the bad reference to AbstractFunctionMojo